host-spec/ae-runtimeapi.tm

```diff
@@ -514,12 +514,15 @@
 
   <subsubsection|<verbatim|TaggedTransactionQueue_validate_transaction>><label|sect-rte-validate-transaction>
 
-  This entry is invoked against extrinsics submitted through the transaction
-  network message <reference|sect-msg-transactions> and indicates if the
-  submitted blob represents a valid extrinsics applied to the specified
-  block. This function gets called internally when executing blocks with the
-  <verbatim|Core_execute_block> runtime function as described in section
-  <reference|sect-rte-core-execute-block>.
+  This entry should be invoked with all extrinsics submitted through a
+  transaction network message <reference|sect-msg-transactions> or by an
+  offchain worker through the <verbatim|ext_offchain_submit_transaction> Host
+  API (Section <reference|sect-ext-offchain-submit-transaction>). It will
+  indicates if the submitted blob represents a valid extrinsics applied to
+  the specified block, how it depends on other extrinsics and if it should be
+  gossiped to other peers. Furthermore this function gets called internally
+  when executing blocks with the <verbatim|Core_execute_block> runtime
+  function as described in section <reference|sect-rte-core-execute-block>.
```

Comment:
> It will indicates ...

should be:

> It will **indicate** ...

But personally I'd write it as:

> It indicates whether ...

Also, I don't understand this part:

> ..., how it depends on other extrinsics ...

Can you clarify what you mean?

Reply:
I replaced it with "in which order it should be applied" to make it more clear what I mean with "depend on"